recipes/miktex-portable/post-link.bat

```diff
@@ -0,0 +1,5 @@
+rem packages for nbconvert
```

Comment:
?

As far as I understand it, these are latex packages, which would get installed when nbconvert calls pdflatex (via pandoc?) when it translates a notebook to a PDF